apps/site/src/app/(home)/sections/Intro/Intro.module.scss

```diff
@@ -2,28 +2,17 @@
 
 .description {
 	text-align: center;
-	font-size: 0.85rem;
+	font-size: 0.75rem;
 
-	@media screen and (min-width: 415px) {
+	@include bootstrap.media-breakpoint-up(sm) {
 		font-size: 1rem;
 	}
-
-	@media screen and (min-width: 500px) {
-		font-size: 1.25rem;
-	}
-
-	@include bootstrap.media-breakpoint-up(xl) {
-		font-size: 1.5rem;
-	}
```

Comment:
Thought: the font size of the What is ZotHacks? description is a bit small on larger screens in my opinion. Did you delete these lines because you think the description text looks better?

Reply:
I think I might've deleted them on accident while removing the margin-top. Added them back